src/v2/view-builder/views/password/ResetAuthenticatorPasswordView.js

```diff
@@ -13,10 +13,6 @@ const Body = EnrollAuthenticatorPasswordView.prototype.Body.extend({
     return loc('password.reset', 'login');
   },
 
-  getPasswordPolicy () { 
```

Comment:
The reason why I left it there is because if in `EnrollAuthenticatorPaswordView` the way we retrieve password policy changes, it will break here..